.travis.yml

```diff
@@ -2,7 +2,7 @@ language: ruby
 dist: precise
 
 rvm:
-  - 2.3.1
+  - 2.6.0
```

Comment:
This also requires update ruby at production server. We should schedule DevOps task,